spec/fortnox/api/repositories/customer_spec.rb

```diff
@@ -22,10 +19,11 @@
                    'test@example.com'
 
   # It is not yet possible to delete Customers. Therefore, expected nr of
-  # Customers when running .all will continue to increase.
+  # Customers when running .all will continue to increase
+  # (until 100, which is max by default).
   include_examples '.all', 100
```

Comment:
I had a thought about this before, unrelated to this PR but I'm telling you now since I remember :) :

I know you can make some tests in rspec sequential. We should change this to be `.all`, save the number of customers, `.new`, `.all` again and compare to the saved number. That way we get dynamic tests without hardcoded values that we have to update.